docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,159 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into several topics. Many
+statements are part of the core engine and therefore available in all use cases.
+For example, you can always set session properties or inspect an explain plan
+and perform other actions with the :ref:`globally available statements
+<sql-globally-available>`.
+
+However the details and architecture of the connected data sources may prevent
+some SQL functionality. For example, if the data source does not support any
+write operations, then a :doc:`/sql/delete` cannot be executed against the data
```

Comment:
/sql/delete` statement cannot be executed

Reply:
done